airflow/providers/salesforce/hooks/tableau.py

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from tableauserverclient import Pager, Server, TableauAuth
+
+from airflow.hooks.base_hook import BaseHook
+
+
+class TableauHook(BaseHook):
+    """
+    Connects to the Tableau Server Instance and allows to communicate with it.
+
+    :param site_id: The id of the site where the workbook belongs to.
+    :type site_id: str
+    :param tableau_conn_id: The Tableau Connection id containing the credentials
+        to authenticate to the Tableau Server.
+    :type tableau_conn_id: str
+    """
+
+    def __init__(self, site_id='', tableau_conn_id='tableau_default'):
```

Comment:
Shouldn't `tableau_default` be created in `airflow/utils/db.py` ?

Reply:
Actually I think I will remove the default
```suggestion
    def __init__(self, site_id='', tableau_conn_id'):
```

Reply:
I think it should have default. It can be useful for example dags. Especially if this connection will require settings in the Extra field (like discussed on the default site)